Automattic/simplenote-ios
# Update CircleCI config for XCode 11

This PR updates the CircleCI configuration to use XCode 11 containers as we're going to release 4.10 built on it. 

### Test
1. Check code changes.
2. Make sure CircleCI is green.

These changes do not require release notes.
